src/DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -67,7 +66,7 @@ public function process(ContainerBuilder $container): void
 
                 $this->replaceDefaultArguments([
                     0 => $id,
-                    2 => CRUDController::class,
+                    2 => 'sonata.admin.controller.crud',
```

Comment:
I find weird the fact we stop using a FQCN class. Could you explain ?

Reply:
This was the old method of loading routes, When you declare your controller as services, you must specify the id when referencing routes:

old:

No service definition + FQCN

new:

Service definition on DI + use the id

There are cases (on application mostly) where you define your services using the FQCN, so the id is the same as the class name. This is already discussed for Symfony as a bad practice on bundles. I can't find the link now, but we have discussed it already on other PRs.